### PR TITLE
Feat/#101 intro screen

### PR DESCRIPTION
**Pull request by telmodias**
Description
-----------
Updated Role Info screen with new graphics, updated Portuguese translations for this screen and added dependency for ant design icons

Motivation
----------

Testing Guidelines
------------------

Release Checklist
-----------------

- [ ] This code has unit tests
- [ ] I have a plan to verify that these changes are working as expected after deploy
- [ ] I have a plan for how to revert, rollback, or disable these changes if things are not working as expected

Security Checklist
-------------------

This PR creates, modifies, or deletes:

- [ ] API routes: API routes, parameters, or user authorization
- [ ] Authentication: Authentication mechanism
- [ ] Credentials: Server side credentials, or secrets in configuration / source code
- [ ] Cryptography: Encryption, hashing, certificates, signatures, random numbers, etc.
- [ ] User data: personal information handling, logs, error messages, etc.

Additional Notes
----------------

### CODE REVIEW

File: web-client/src/pages/role.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (micaelr95) [inline web-client/src/pages/role.tsx:1]: Rename the file from role.tsx to RolePage.tsx to be consistent with the rest of the project.

File: web-client/src/components/RoleInfo/RoleInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+// const Info = styled(Text)`
+//   margin-top: 40px;
+//   text-align: center;
+//   font-size: 20px;
+// `;
+
+// const Coll = styled(Col)`
+//   text-align: center;
+// `;
```
Comment (micaelr95) [inline web-client/src/components/RoleInfo/RoleInfo.tsx:39]: Is this to use later or you just forgot to delete?

File: web-client/src/pages/role.tsx (hunk 1 above)
Comment (telmodias) [reply web-client/src/pages/role.tsx:1]: Renamed to RoleInfoPage.tsx

File: web-client/src/components/RoleInfo/RoleInfo.tsx (hunk 2 above)
Comment (telmodias) [reply web-client/src/components/RoleInfo/RoleInfo.tsx:39]: Forgot to delete. Fixed.

**Comment by telmodias** [comment]
Created a new PR with suggested changes.

**Comment by micaelr95** [comment]
@all-contributors please add @micaelr95 for review.

**Pull request closed without merging**